packages/vm/src/index.ts

```diff
@@ -288,7 +283,7 @@ export default class VM extends AsyncEventEmitter {
     if (opts.stateManager) {
       this.stateManager = opts.stateManager
     } else {
-      const trie = opts.state ?? new Trie()
+      const trie = new Trie()
```

Comment:
Side note, not directly related: I guess it would rather make sense, to make `trie` optional for the `StateManager` options and then do this trie instantiation there if not provided.

Reply:
that makes sense, i should have checked, we already do that :) so i can just remove this, i will make a note for myself to do in a follow up develop PR